jest.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
 	moduleDirectories: ['node_modules'],
+	testEnvironment: 'jsdom',
```

Comment:
Question: is this necesary? this add jsdom as default environment to all test, that means that all the test going to have all the dom properties, using more amount of ram and can broke the test in ci, you can specify the jsdom environment just in the particular test class that need it

Reply:
[Comment]  tests are not passing if `testEnvironment` is not defined as jsdom and that is because i upgraded the version of jest to the latest version v29 which has a breaking change for that from v27.